mne/epochs.py

```diff
@@ -1252,7 +1252,10 @@ def drop(self, indices, reason='USER', verbose=None):
         keep = np.setdiff1d(np.arange(len(self.events)), try_idx)
         self._getitem(keep, reason, copy=False, drop_event_id=False)
         count = len(try_idx)
-        logger.info('Dropped %d epoch%s' % (count, _pl(count)))
+        logger.info('Dropped %d epoch%s: %s' % (count, _pl(count), 
```

Comment:
@hoechenberger: how about this part (formatting and array to string) – is there a more efficient way to do it?